concent-builder/files/jenkins/github-plugin-configuration.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version='1.1' encoding='UTF-8'?>
+<github-plugin-configuration plugin="github@1.29.3">
+  <configs/>
+  <hookUrl>http://builder.concent.golem.network/github-webhook/</hookUrl>
```

Comment:
The domain (or even the whole URL) should come from concent-deployment-values and not be hard-coded.